cfme/fixtures/authentication.py

```diff
@@ -19,6 +19,7 @@ def setup_aws_auth_provider(appliance, amazon_auth_provider):
     original_config = appliance.server.authentication.auth_settings
     appliance.server.authentication.configure(auth_mode='amazon',
                                               auth_provider=amazon_auth_provider)
+    sleep(60)  # wait for MIQ to update, no trigger to look for, but if you try too soon it fails
```

Comment:
Q: is there something to check instead of sleep ?

Reply:
There isn't, but I think I can drop this to 30s